client/lib/explat/internals/log-error.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * WordPress dependencies
+ */
+// TODO: Replace module in browser SSR context
+import { getLogger } from 'calypso/server/lib/logger';
+
+// Using typescript to ensure we are being safe in SSR
+declare const window: undefined | ( Window & typeof globalThis );
+
+export const isDevelopmentMode = process.env.NODE_ENV === 'development';
+
+/**
+ * Log an error to the server.
+ * Works in SSR.
+ *
+ * Should never throw.
+ *
+ * @param error Error to save
+ */
+export const logError = ( error: Record< string, string > & { message: string } ): void => {
+	try {
+		if ( isDevelopmentMode ) {
+			// eslint-disable-next-line no-console
+			console.error( '[ExPlat]', error.message, error );
```

Comment:
I thin than when running in SSR, this will cause a duplicated error (one here, another one when calling `getLogger()`). Maybe do this only when running in the browser?

Reply:
Sometimes I do run it in dev mode :) It will generate two console logs, one in plain form from `console.error` and a second one in JSON form from bunyan. Not a huge deal anyway, just a small nit.

Reply:
No problem, it should be fixed :-)